### PR TITLE
[iOS] fixed MAPSME-9555 app is crashing when open through review push.

### PR DESCRIPTION
**Pull request by niakrise**
https://jira.mail.ru/browse/MAPSME-9555

### CODE REVIEW

File: iphone/Maps/Classes/Components/ExpandableTextView/ExpandableTextView.swift [hunk 1]
```diff
@@ -1,184 +1,120 @@
...
-    super.awakeFromNib()
-    setup()
-  }
-
-  public convenience init() {
-    self.init(frame: CGRect())
-  }
-
-  public override init(frame: CGRect) {
+final class ExpandableReviewView: UIView {
```
Comment (beloal) [inline iphone/Maps/Classes/Components/ExpandableTextView/ExpandableTextView.swift]: Please rename the file as well

File: iphone/Maps/Classes/Components/ExpandableTextView/ExpandableTextView.swift [hunk 2]
```diff
@@ -1,184 +1,120 @@
...
+  }()
+  
+  var moreLabel: UILabel = {
+    let label = UILabel(frame: .zero)
+    label.clipsToBounds = true
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+  
+  var moreZeroHeight: NSLayoutConstraint!
```
Comment (beloal) [inline iphone/Maps/Classes/Components/ExpandableTextView/ExpandableTextView.swift]: I'd name it `moreLabelZeroHeight`

File: iphone/Maps/Classes/Components/ExpandableTextView/ExpandableTextView.swift [hunk 3]
```diff
@@ -1,184 +1,120 @@
...
+    super.layoutSubviews()
+    updateRepresentation()
+  }
+  
+  func configureContent() {
+    self.addSubview(contentLabel)
+    self.addSubview(moreLabel)
+    let labels: [String: Any] = ["contentLabel": contentLabel, "moreLabel": moreLabel]
+    var contentConstraints: [NSLayoutConstraint] = []
+    let verticalConstraints = NSLayoutConstraint.constraints(withVisualFormat: "V:|-0-[contentLabel]-0-[moreLabel]",
```
Comment (beloal) [inline iphone/Maps/Classes/Components/ExpandableTextView/ExpandableTextView.swift]: Is there any reason to specify zero spacing `-0-` explicitly?

Comment (niakrise) [reply iphone/Maps/Classes/Components/ExpandableTextView/ExpandableTextView.swift]: Yes, otherwise it will apply the default margins (8pt for our case).

**Pull request merged**